lighthouse-extension/gulpfile.js

```diff
@@ -118,7 +118,8 @@ gulp.task('browserify-lighthouse', () => {
       .ignore('source-map')
       .ignore('whatwg-url')
       .ignore('url')
-      .ignore('debug/node');
+      .ignore('debug/node')
+      .ignore('pako/lib/zlib/inflate.js');
```

Comment:
I might be looking wrong but I don't see it being injected.
![image](https://cloud.githubusercontent.com/assets/1120926/24928014/35c5d2ba-1f01-11e7-865d-6a420a1a8323.png)